helm-chart/renku-graph/values.yaml

```diff
@@ -130,6 +135,7 @@ triplesGenerator:
   ## a demanded number of concurrent triples generation processes
   generationProcessesNumber: 2
   transformationProcessesNumber: 2
+  connectionPool: 2
```

Comment:
I'm guessing whether a connection pool of 2 would be good enough for TG. A single connection shouldn't be in use by the lock for long but assuming there are several different events being processed at the same time and some of them might be polling for a lock for a bit, two connections might be tight. But on the other side, maybe it's a good value for now. Just wondering how we'll find out if the setting is fine.

Reply:
Yes, you are right - I didn't think much when copying these sections, tbh. Perhaps a bit higher value, like 5 or 10? would be better